src/components/InviteLandingPage.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { useLocation, Navigate, useParams } from 'react-router-dom';
+import { PropTypes, instanceOf } from 'prop-types';
+import Register from './register/register';
+import { NPOBackend } from '../utils/auth_utils';
+import { Cookies, withCookies } from '../utils/cookie_utils';
+
+const InviteLandingPage = () => {
+  const [invite, setInvite] = useState();
+  const [validInvite, setValidInvite] = useState(true);
+  const [firstName, setFirstName] = useState();
+  const [lastName, setLastName] = useState();
+  const [email, setEmail] = useState();
+  const [role, setRole] = useState();
+  const [isLoading, setIsLoading] = useState();
+  const { search } = useLocation();
+  const { inviteID } = useParams();
+
+  const retrieveInvite = async id => {
+    console.log('retrieveInvite() called in InviteLandingPage');
+    const foundInvite = await NPOBackend.get(`/adminInvite/${id}`);
+    console.log('foundInvite looks like:');
+    console.log(foundInvite);
+    if (!inviteID || !foundInvite || !foundInvite.data) {
+      setValidInvite(false);
+    } else if (JSON.stringify(foundInvite.expireDate) < JSON.stringify(Date())) {
+      setValidInvite(false);
+    } else {
+      console.log('setting fname, lname, email, and role in state');
+      setFirstName(foundInvite.data.firstName);
+      setLastName(foundInvite.data.lastName);
+      setEmail(foundInvite.data.email);
+      setRole(foundInvite.data.role);
+    }
+  };
+
+  const useEffect = async () => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useEffect' is assigned a value but never used.